frontend/public/locales/en/translation.json

```diff
@@ -409,6 +410,18 @@
+    "entertaining_relaxing": "<0>Do people feel good watching it?</0><1>Contents that score high on \"entertaining and relaxing\" should entertain and relax viewers.</1>",
+    "importance": "<0>Can additional focus on this topic have a significantly positive impact on the world?</0><1>This criterion aims to combat the problem of <1>mute news</1>, which are information that do not have the attention that they should have for the good of humanity and beyond. As an example, as explained by <4>this PBS Hot Mess video</4>, when it comes to mitigating climate change, asking viewers to switch off the light when not at home is orders of magnitude less important than, say, avoid flying to countries far away.</1><2>Contents that score high on \"important and actionable\" should present data and arguments with major consequences, as well as actionable plans that would have large-scale mpacts.</2><3>Scale</3><4>Plans with large-scale impacts are more important, as they affect more individuals at once.</4><5>Actionable and tractable</5><6>A content that simply lists problem is arguably less important to share than a content that both lists problems and propose actionable solutions.</6><7>The content is all the more important if it calls for actionable plans from the viewers (even though these plans may consist in creating social pressure on other individuals).</7><8>Other aspects</8><9>When judging the importance of a content, it may be relevant to reflect on whether the content stresses the important features of the topic it discusses. In particular, introductions and conclusions may be what catches viewers' attention the most. Does the content open and close with important considerations?</9><10>According to Bayesianism, <1>surprising</1> data ought to have a much larger impact on a (Bayesian) viewer's beliefs. In practice, viewers are not Bayesian. This is why, in addition to presenting surprising data to viewers, it is also important to also stress the surprisingness of the data or argument, by highlighting it. This means underlining the neglectedness of the data or argument.</10>",
```

Comment:
```suggestion
    "importance": "<0>Can additional focus on this topic have a significantly positive impact on the world?</0><1>This criterion aims to combat the problem of <1>mute news</1>, which are information that do not have the attention that they should have for the good of humanity and beyond. As an example, as explained by <4>this PBS Hot Mess video</4>, when it comes to mitigating climate change, asking viewers to switch off the light when not at home is orders of magnitude less important than, say, avoid flying to countries far away.</1><2>Contents that score high on \"important and actionable\" should present data and arguments with major consequences, as well as actionable plans that would have large-scale impacts.</2><3>Scale</3><4>Plans with large-scale impacts are more important, as they affect more individuals at once.</4><5>Actionable and tractable</5><6>A content that simply lists problem is arguably less important to share than a content that both lists problems and propose actionable solutions.</6><7>The content is all the more important if it calls for actionable plans from the viewers (even though these plans may consist in creating social pressure on other individuals).</7><8>Other aspects</8><9>When judging the importance of a content, it may be relevant to reflect on whether the content stresses the important features of the topic it discusses. In particular, introductions and conclusions may be what catches viewers' attention the most. Does the content open and close with important considerations?</9><10>According to Bayesianism, <1>surprising</1> data ought to have a much larger impact on a (Bayesian) viewer's beliefs. In practice, viewers are not Bayesian. This is why, in addition to presenting surprising data to viewers, it is also important to also stress the surprisingness of the data or argument, by highlighting it. This means underlining the neglectedness of the data or argument.</10>",
```